xml/wgl.xml

```diff
@@ -72,44 +72,44 @@ Registry at https://github.com/KhronosGroup/OpenGL-Registry
          sometimes reused for other purposes -->
 
```

Comment:
Note this comment (was originally in gl.xml but must've got removed):
```xml

    <!-- SECTION: GL parameter class type definitions.
         The groups are intended to contain all the possible legal values
         for corresponding function parameters, but it is likely that many
         of the groups are out of date relative to current OpenGL and OpenGL
         ES specifications, and the many extensions to those specifications.
         As such, they may not be a reliable source for enumeration info.

         We welcome assistance from the community in achieving and
         maintaining the completeness of the enum groups. Khronos does not
         use the enum group information, and the OpenGL Working Group does
         not have internal resources to bring it up to date.
    -->
```

This was added in #280 and #291. While thanks to community efforts this situation has got a lot better, I do think it is worth adding a comment in both files that the groups at the very least **are exclusively community-supported, and that the OpenGL Working Group has no intention of keeping the groupings up-to-date themselves due to them not being relevant for Khronos uses**. It's annoying we missed this comment being removed from gl.xml.

Reply:
We can add this comment to these files too.